Pytest failed for tqdm version 4.41.1

When running pytest on my machine I got an error regarding the tqdm module:
```
==================================== ERRORS ====================================
______________ ERROR collecting tests/data/test_LightlyDataset.py ______________
ImportError while importing test module '/home/igor_lightly_ai/lightly/tests/data/test_LightlyDataset.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/conda/envs/lightly/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/data/test_LightlyDataset.py:12: in <module>
    from lightly.data._utils import check_images
lightly/data/_utils.py:10: in <module>
    import tqdm.contrib.concurrent as concurrent
E   ModuleNotFoundError: No module named 'tqdm.contrib.concurrent'
```

When uninstalling tqdm and installing the latest version (4.56.2) through pip it worked.

--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -5,7 +5,7 @@ numpy>=1.18.1
 pytorch_lightning>=1.0.4
 requests>=2.23.0
 torchvision
-tqdm
+tqdm>=4.44
 certifi >= 14.05.14
 six >= 1.10
 python_dateutil >= 2.5.3

